sdcm/cluster_docker.py

```diff
@@ -191,15 +223,19 @@ def restart_scylla_server(self, verify_up_before=False, verify_up_after=True, ti
         if verify_up_before:
             self.wait_db_up(timeout=verify_up_timeout)
 
-        # Need to restart the scylla-housekeeping service manually because of autostart of this service is disabled
-        # for the docker backend. See, for example, docker/scylla-sct/ubuntu/Dockerfile
-        self.stop_scylla_housekeeping_service(timeout=timeout)
-        # ignoring WARN messages upon stopping - https://github.com/scylladb/scylla-cluster-tests/issues/10633
-        with DbEventsFilter(db_event=DatabaseLogEvent.BACKTRACE, line="WARN "):
-            self.remoter.sudo('sh -c "{0} || {0}-server"'.format("supervisorctl restart scylla"), timeout=timeout)
+        # It's likely that we won't need this step after scylladb docker image becomes supervisor-less
+        # # Need to restart the scylla-housekeeping service manually because of autostart of this service is disabled
+        # # for the docker backend. See, for example, docker/scylla-sct/ubuntu/Dockerfile
+        # self.stop_scylla_housekeeping_service(timeout=timeout)
+        if self.docker_service_manager:
+            self.docker_service_manager.restart_service('scylla', timeout=timeout)
+        else:
+            # ignoring WARN messages upon stopping - https://github.com/scylladb/scylla-cluster-tests/issues/10633
+            with DbEventsFilter(db_event=DatabaseLogEvent.BACKTRACE, line="WARN "):
+                self.remoter.sudo('sh -c "{0} || {0}-server"'.format("supervisorctl restart scylla"), timeout=timeout)
         if verify_up_after:
             self.wait_db_up(timeout=verify_up_timeout)
-        self.start_scylla_housekeeping_service(timeout=timeout)
+        # self.start_scylla_housekeeping_service(timeout=timeout)
```

Comment:
we can remove anything related to housekeep, I'm o.k. for not test housekeep at all with docker backend